200-Storage-Cross-Account-S3-Copy/README.MD

```diff
@@ -74,7 +75,10 @@ Your client has a team of developers using `Account A` to create assets to be us
         }
     ]
 }
-        ```
+
+ ```
+ 
+ 
     **Note:** _The Source account's ACCOUNT NUMBER can be figured by going to the SOURCE ACCOUNT's CONSOLE, then click SUPPORT, choose Support center and there you     will find the account number as : `xxxxxxxxxxxx`_
 
  1. ## Verify S3 Copying & Ownership:
```

Comment:
The numbering is not correct, since the formatting is not offset correctly

Reply:
Can you write a small intro into your solution on how you are planning to achieve the ownership transfer.